visgraph/graphcore.py

```diff
@@ -591,8 +612,8 @@ class HierGraph(Graph):
     NOTE: rootnodes are designated by the presence of the "rootnode"
           property.
     '''
-    def __init__(self):
-        Graph.__init__(self)
+    def __init__(self, merge=None):
+        Graph.__init__(self, merge)
```

Comment:
plz always pass named arguments with their names to prevent order changes or argument addtion/removal from breaking the above code ( in this case, merge=merge )